charts/sdf-generation/templates/sdf-generation.yaml

```diff
@@ -0,0 +1,66 @@
+{{- $name := include "sdf-generation.fullname" . -}}
+
+
+apiVersion: apps/v1
+kind: Deployment 
+metadata:
+  name: {{ include "sdf-generation.fullname" . }}-worker
+  labels:
+    {{- include "sdf-generation.labels" . | nindent 4 }}
+spec:
+  {{- if not .Values.autoscaling.enabled }}
+  replicas: {{ .Values.replicaCount }}
+  {{- end }}
+  selector:
+    matchLabels:
+      {{- include "sdf-generation.selectorLabels" . | nindent 6 }}
+      role: worker
+  template:
+    metadata:
+      annotations:
+        checksum/config-map: {{ include (print $.Template.BasePath "/configmap.yaml") . | sha256sum }}
+      labels:
+        {{- include "sdf-generation.selectorLabels" . | nindent 8 }}
+        role: worker
+    spec:
+      serviceAccountName: {{ include "sdf-generation.serviceAccountName" . }}
+      {{- with .Values.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      containers:
+      - name: {{ .Chart.Name }}
+        image:  "{{ .Values.image.repository }}:{{ .Values.image.tag }}" 
+        imagePullPolicy:  {{ .Values.image.pullPolicy }}
+        ports:
```

Comment:
Since your app is just a worker consumer I would remove this entire port section since your app won't be operating on any ports.

Reply:
Its been removed